README.md

```diff
@@ -14,20 +40,66 @@ IRC channel: [freenode](https://freenode.net) #googlebenchmark
 
 [Assembly Testing Documentation](docs/AssemblyTests.md)
 
+## Requirements
+
+Google Benchmark uses C++11 when building the library. As such we require
+a modern C++ toolchain, both compiler and standard library.
+
+The following minimum versions are strongly recommended build the library:
+
+* GCC 4.8
+* Clang 3.4
+* Visual Studio 2013
+* Intel 2015 Update 1
+
+Anything older *may* work.
+
+Note: Using the library and its headers in C++03 is supported. C++11 is only
+required to build the library.
 
-## Building
+## Installation
 
-The basic steps for configuring and building the library look like this:
+This describes the installation process using cmake. As pre-requisites, you'll
+need git and cmake installed.
 
 ```bash
+# Check out the library.
 $ git clone https://github.com/google/benchmark.git
 # Benchmark requires Google Test as a dependency. Add the source tree as a subdirectory.
 $ git clone https://github.com/google/googletest.git benchmark/googletest
+# Make a build directory to place the build output.
 $ mkdir build && cd build
-$ cmake -G <generator> [options] ../benchmark
-# Assuming a makefile generator was used
+# Generate a Makefile with cmake.
+# Use cmake -G <generator> to generate a different file type.
+$ cmake ../benchmark
+# Compile the library.
 $ make
 ```
+You should now see `libbenchmark.a` and `libbenchmark_main.a` under the
```

Comment:
This is extremely specific.
Is that *nix-only? Wouldn't this differ on mac? windows?
This also assumes that benchmark was built as static library, not a shared one.

I'm not sure why this new section is needed.

Reply:
> Is that *nix-only? Wouldn't this differ on mac? windows?

True. I'll revise this.

> I'm not sure why this new section is needed.

Just to clarify the libraries' names and locations after following the build instructions. It took me a bit of fiddling to find them after installation. I initially confused them with `libgtest` and `libgtest_main`.